src/Symfony/Bridge/PhpUnit/Legacy/ForwardCompatTestTraitForV5.php

```diff
@@ -79,4 +79,114 @@ private function doTearDown()
     {
         parent::tearDown();
     }
+
+    /**
+     * @param string $message
+     *
+     * @return void
+     */
+    public static function assertIsArray($actual, $message = '')
```

Comment:
```suggestion
    public static function assertIsArray($actual, string $message = '')
```

should be no problem, right?

Reply:
not compatible with php 5. The bridge don't use the same version of PHP.

https://github.com/symfony/symfony/blob/f8302262714674e3efbd775c9b107294d89fb77e/src/Symfony/Bridge/PhpUnit/composer.json#L18-L22

Reply:
Thanks!